src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageClientMiddleLayer.cs

```diff
@@ -17,70 +13,16 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor
     [Export(typeof(RazorLanguageClientMiddleLayer))]
     internal class DefaultRazorLanguageClientMiddleLayer : RazorLanguageClientMiddleLayer
     {
-        private readonly LSPDocumentManager _documentManager;
-        private readonly LSPEditorService _editorService;
-
-        [ImportingConstructor]
-        public DefaultRazorLanguageClientMiddleLayer(LSPDocumentManager documentManager, LSPEditorService editorService)
-        {
-            if (documentManager is null)
-            {
-                throw new ArgumentNullException(nameof(documentManager));
-            }
-
-            if (editorService is null)
-            {
-                throw new ArgumentNullException(nameof(editorService));
-            }
-
-            _documentManager = documentManager;
-            _editorService = editorService;
-        }
-
-        public override bool CanHandle(string methodName)
-        {
-            return methodName == Methods.TextDocumentOnTypeFormattingName;
-        }
+        public override bool CanHandle(string methodName) => false;
```

Comment:
Hmm, I kind of liked having the "empty" version of this stay around as an example until I saw that it always returned false from `CanHandle`.

Reply:
Ya I figured it'd be a decent thing to keep for now in the case we need to use it again